### PR TITLE
chore(CI) : Update workflow name  

### PR DESCRIPTION
**Pull request by MedWassCh**
## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x
- [ ] 24.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by kduret — COMMENTED** [review]
all workflow runs will be nightly ? :)

**Review by paul-oureib — DISMISSED** [review]

File: .github/workflows/web.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
-name: web
+name: dev-23.10.X
+run-name: web dev-23.10.X ${{ github.head_ref || github.ref_name }}
```
Comment (kduret) [inline .github/workflows/web.yml:2]:
```suggestion
name: web
run-name: ${{ github.event_name == 'schedule' || (github.event_name == 'dispatch' && inputs.is_nightly == 'true') && format('web nightly {0}', github.ref_name) }}
```

**Review by kduret — COMMENTED** [review]
Did you test a manuel trigger of web workflow with those changes ?

File: .github/workflows/web.yml [hunk 2]
```diff
@@ -1,4 +1,6 @@
 name: web
+run-name: ${{ github.event_name == 'schedule' || (github.event_name == 'dispatch' && inputs.is_nightly == 'true') && format('web nightly {0}', github.ref_name) }}
```
Comment (kduret) [inline .github/workflows/web.yml:2]: I'm not sure, but maybe dispatch should be replaced by workflow_dispatch

Comment (MedWassCh) [reply .github/workflows/web.yml]: image
with the actual update we have this output , but while adding && inputs.is_nightly == 'true' we will get false is it necessary to adding it

File: .github/workflows/web.yml [hunk 3]
```diff
@@ -1,4 +1,6 @@
 name: web
+run-name: ${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch') && format('web nightly {0}', github.ref_name) }}
```
Comment (kduret) [inline .github/workflows/web.yml:2]: where is the usage of input `is_nightly` ? :(

**Review by kduret — CHANGES_REQUESTED** [review]

File: .github/workflows/web.yml (hunk 3 above)
Comment (MedWassCh) [reply .github/workflows/web.yml:2]: image

with the actual update we have this output , but while adding && inputs.is_nightly == 'true' we will get false is it necessary to adding it?

Comment (kduret) [reply .github/workflows/web.yml:2]: this is because you need to add `|| ''` at the end of the condition

Comment (kduret) [reply .github/workflows/web.yml:2]: check here : https://github.com/centreon/centreon/pull/2872/files#diff-5d8adc0ecc6d4ffce45387b7523fb6cca8539efedfeff49cac7a722a10c8160cR2

**Review by kduret — APPROVED** [review]

**Pull request merged**